falcon/util/__init__.py

```diff
@@ -20,9 +20,18 @@
 
 """
 
+from http import cookies as http_cookies
 import json  # NOQA
 
 # Hoist misc. utils
 from falcon.util.structures import *  # NOQA
 from falcon.util.misc import *  # NOQA
 from falcon.util.time import *  # NOQA
+
+
+# NOTE(kgriffs): Backport support for the new 'SameSite' attribute
+#   for Python versions prior to 3.8. We do it this way because
+#   SimpleCookie does not give us a simple way to specify our own
+#   subclass of Morsel.
+if 'samesite' not in http_cookies.Morsel._reserved:  # pragma: no cover
```

Comment:
I'm thinking whether we should somehow propagate this warning into our docs that we monkey-patch this way.
I don't really see this backport doing any harm, but OTOH it's hard to exhaust all possible things people are doing with the standard lib.

But I'm not insisting on this change for merging this PR. We can do this in a follow-up PR if we decide to.

Reply:
Not a bad idea to at least call it out in the docs.